.github/workflows/cli-release.yml

```
@@ -114,7 +116,8 @@ jobs:
       attestations: write      # Allows storing provenance attestation
       artifact-metadata: write # Needed for github artifact metadata publishing for oci
     uses: ./.github/workflows/cli.yml
-    secrets: inherit
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] secret "GITHUB_TOKEN" is not defined in "./.github/workflows/cli.yml" reusable workflow. no secret is defined

(workflow-call)